pulsar-broker/src/test/java/org/apache/pulsar/broker/service/AbstractBaseDispatcherTest.java

```diff
@@ -214,6 +211,10 @@ protected boolean isConsumersExceededOnSubscription() {
 
         @Override
         public boolean trackDelayedDelivery(long ledgerId, long entryId, MessageMetadata msgMetadata) {
+            if (!msgMetadata.hasDeliverAtTime()) {
```

Comment:
this change seems unrelated to the patch

Reply:
it relates to the entry filters unit test.

Reply:
I will submit a new pr to solve this.